packages/meteor-gazelle-wiki/lib/templates/article.html

```diff
@@ -0,0 +1,7 @@
+<template name="article">
+  {{ title }}
+  {{ sections }}
+  {{ aliases }}
+  {{ createdAt }}
+  {{ updatedAt }}
+</template>
```

Comment:
Certainly. But routing isn't working in our app, yet, (and I've tried and failed to fix it,) so I'm unable to work more on the templates until it is.